rotkehlchen/assets/utils.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Optional, Union
+
+from rotkehlchen.errors import UnknownAsset
+from rotkehlchen.typing import ChecksumEthAddress
+
+from .asset import EthereumToken
+from .unknown_asset import UnknownEthereumToken
+
+
+def get_ethereum_token(
+        symbol: str,
+        ethereum_address: ChecksumEthAddress,
+        name: Optional[str] = None,
+        decimals: Optional[int] = None,
+) -> Union[EthereumToken, UnknownEthereumToken]:
+    """Given a token symbol and address return the <EthereumToken>, otherwise
+    an <UnknownEthereumToken>.
+    """
+    ethereum_token: Any = None
+    is_unknown_asset = False
+    try:
+        ethereum_token = EthereumToken(symbol)
+    except UnknownAsset:
+        is_unknown_asset = True
+    else:
+        if ethereum_token.ethereum_address != ethereum_address:
```

Comment:
At the beginning I had the log placed here, but as I wanted to make it a "utils" I placed the log into `_get_balances_graph` (if the asset is an instance of `UnknownEthereumToken`). If you want it here it is ok.

Reply:
Hm I would prefer log entries as early as possible as you never know how this function may be used in the future. But I won't insist too much on it since for now you use it in only that one place.